pkg/job/runtime/kubernetes/executor/vcjob.go

```diff
@@ -37,6 +37,7 @@ const (
 
 type VCJob struct {
 	KubeJob
+	Flavour schema.Flavour
```

Comment:
It might be better to get `Flavour` from tasks for single pod.

Reply:
removed